test/onnx/test_fx_op_consistency.py

```diff
@@ -136,11 +137,18 @@
         # "nn.functional.scaled_dot_product_attention"  non-deterministic
         "scatter_add",
         "scatter_reduce",
+        "stft",
```

Comment:
Should `stft` be put in `COMPLEX_TESTED_OPS`?

Reply:
Sorry, this PR needs registry updated. I will turn it to draft for now.